lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -5,6 +5,7 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 @import '../../scss-shared/_nx-form-validation.scss';
+@import '../../scss-shared/_nx-text-helpers.scss';
```

Comment:
Is this import actually needed?

Reply:
> Is this import actually needed?

No, apparently it is not: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/bacb658dffb98e510909b16c3405b6963ede1c2b